pkg/manager/manager.go

```diff
@@ -407,6 +416,14 @@ func (o Options) AndFrom(loader config.ControllerManagerConfiguration) (Options,
 		o.CertDir = newObj.Webhook.CertDir
 	}
 
+	if o.Controller.CacheSyncTimeout == nil && newObj.Controller.CacheSyncTimeout != nil {
```

Comment:
should we check if `o.Controller` is `nil` first?

Reply:
o.Controller is a non-pointer, to avoid checking it here

Reply:
oh gotcha, confused with the exported controllerconfiguration. makes sense.